proto/osmosis/concentrated-liquidity/tx.proto

```diff
@@ -75,12 +80,34 @@ message MsgCreatePositionResponse {
 message MsgAddToPosition {
   uint64 position_id = 1 [ (gogoproto.moretags) = "yaml:\"position_id\"" ];
   string sender = 2 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
-  cosmos.base.v1beta1.Coin token_desired0 = 3 [
-    (gogoproto.moretags) = "yaml:\"token_desired0\"",
+  // amount0 represents the amount of token0 willing to put in.
+  string amount0 = 3 [
```

Comment:
What was the motivation for changing this from coin to amount? (I believe createPosition does coins for desired input & int for min output)

Reply:
The motivation for this was that we do not use the denom given with the coin anyways, there is no need to take in coins